agent/app/agent_capability.go

```diff
@@ -499,6 +506,40 @@ func (agent *ecsAgent) appendServiceConnectCapabilities(capabilities []*ecs.Attr
 	return capabilities
 }
 
+func (agent *ecsAgent) appendEBSTaskAttachCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
+	// todo update to import multiple daemons and append capabilities
+	// for now load only EBS CSI Driver daemon
+	daemonDefinitions, err := md.ImportAll()
+	if err != nil {
+		logger.Error(fmt.Sprintf("Daemon import failure: %s", err))
+		return capabilities
+	}
+	if len(daemonDefinitions) == 0 {
```

Comment:
Q: Will `daemonDefinitions` ever be empty AND we don't get an error from `md.ImportAll()`?

Reply:
yes -- I'm updating this to be a `logger.Warn` as this is possible if there are no definitions to import.